spring-cloud-dataflow-build/spring-cloud-dataflow-build-dependencies/pom.xml

```diff
@@ -71,6 +214,13 @@
 				<artifactId>commons-compress</artifactId>
 				<version>${commons-compress.version}</version>
 			</dependency>
+			<dependency>
```

Comment:
Same as above...

Only need to set [rsocket.version](https://github.com/spring-projects/spring-boot/blob/5490e73922b37a7f0bdde43eb318cb1038b45d60/spring-boot-project/spring-boot-dependencies/build.gradle#L1499-L1509).